api/gateway/v1alpha1/gateway_types.go

```diff
@@ -98,6 +101,13 @@ func (gw *Gateway) Validate(_ context.Context, _ kclient.Reader) error {
 		return fmt.Errorf("VTEPIP %s must be an IPv4 address", gw.Spec.VTEPIP) //nolint:goerr113
 	}
 
+	if gw.Spec.VTEPMAC == "" {
+		return fmt.Errorf("VTEPMAC must be set") //nolint:goerr113
+	}
+	if _, err := net.ParseMAC(gw.Spec.VTEPMAC); err != nil {
```

Comment:
ideally we would reject multicast macs here.  They aren't legal for this use case and linux won't let us assign them.

Possibly we should require that the mac be locally administered as well given that it is for a virtual interface.  That is debatable tho.

Reply:
May also be a good idea to exclude all zeros as that mac has reserved meaning